great_expectations/core/batch.py

```diff
@@ -628,9 +628,10 @@ def head(self, n_rows=5, fetch_all=False):
         return self._data.execution_engine.resolve_metrics((metric,))[metric.id]
 
 
+# TODO: <Alex>ALEX -- Make this helper utility of general use.</Alex>
 def materialize_batch_request(
     batch_request: Optional[Union[BatchRequestBase, dict]] = None,
-) -> Optional[Union[BatchRequest, RuntimeBatchRequest]]:
+) -> Optional[BatchRequestBase]:
```

Comment:
Did we combine these classes back together or do we not have a use for `RuntimeBatchRequest` in this case?

Reply:
@NathanFarmer The both extend `BatchRequestBase` (it has been like that for a while).

Reply:
I misread `BatchRequestBase` as `BatchRequest` 🙇🏻